composer.json

```diff
@@ -29,6 +28,13 @@
     }
   },
   "require-dev": {
+    "phpunit/phpunit": "^5.2",
```

Comment:
You're right. I rebased the repo and just kept the added test script in the composer.json file ([commit here](https://github.com/martin-helmich/php-mongomock/pull/1/commits/8fc6924fc94b088608bec35b048c440e3661b199))